src/openforms/forms/tests/test_models.py

```diff
@@ -262,3 +262,13 @@ def test_form_definition_sensitive_information_returns_correct_fields(self):
             self.form_definition_with_sensitive_information.sensitive_fields,
             ["textFieldSensitive"],
         )
+
+
+class FormStepTestCase(TestCase):
```

Comment:
Please add tests for `FormStep` instances without `form` or `form_definition` set.